doc/html/en/ref/man1/gfuser.1.html

```diff
@@ -56,11 +64,23 @@
 Deletes a registered user.  This option can be used by Gfarm
 administrators.
 </p></dd>
+<dt><span class="term"><code class="option">-L</code></span></dt>
+<dd><p>
+Displays verbose information and user identifiers for
+each authentication methods of registered users.
+</p></dd>
+<dt><span class="term"><code class="option">-A</code></span></dt>
+<dd><p>
+Creates, modifies and deletes user identifiers for each authentication
+methods mapping to global username.
+When deleting, specifies null string for a authentication identifier.
+This option can be used by Gfarm administrators.
+</p></dd>
 <dt><span class="term"><code class="option">-?</code></span></dt>
 <dd><p>Displays a list of command options.</p></dd>
 </dl></div>
 </div>
-<div class="refsect1" title="SEE ALSO">
+<div class="refsect1">
```

Comment:
In Gfarm, it is customary practice to separate commits of human-modified files from commits of automatically generated files.
This is because automatic merging may not work well for auto-generated files.

A commit of auto-generated files are made immediately after a commit of human-modified files, with a commit log "regen".
For example,
human-modified files: 7969aab40
auto-generated files: 650d53f37

Thus, please remove auto-generated files from the pull-request.